### ISSUE TITLE
[BITV] 9.1.4.1/3.2 - When adding a table in the editing section, the inserted controls for actions ("Delete this column" etc.) in the column header cells are not visible. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/text#3861]
- [ ] Fix
- [ ] Backport

![738a8ba209ff1461860050a35da24d4c](https://user-images.githubusercontent.com/1315170/222220464-77de281e-d866-4568-b124-adaaed353b28.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-45ae33ca9f-v3-n2

</details>

- [ ] 9.1.4.1/10.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-45ae33ca9f-v10-n2

**Comment by juliushaertl** [comment]
@AndyScherzinger Which version was tested exactly?

This should have been fixed and back ported to 25.0.4 RC1

https://github.com/nextcloud/text/issues/3369
https://github.com/nextcloud/text/pull/3733

Further ref:
https://github.com/nextcloud/server/pull/36462

**Comment by AndyScherzinger** [comment]
@juliushaertl a daily build of 25 -> [Nextcloud Hub 3](https://nextcloud.com/) (25.0.3 Build:2023-01-30T23:44:03+00:00 4bd771f1951d4108daeb0715446ae8eee66e5183)

**Comment by juliushaertl** [comment]
Ok, fix was merged on Feb, 1st.

**Comment by AndyScherzinger** [comment]
> Ok, fix was merged on Feb, 1st.

One reference still open, not sure all got v25 backported, can you check and if everything is in, close it? Thanks in advance

**Comment by juliushaertl** [comment]
Yep, will retest to be sure
